save-common/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessBuilder.kt

```diff
@@ -102,10 +105,11 @@ class ProcessBuilder(private val useInternalRedirections: Boolean, private val f
 
         val cmd = modifyCmd(command, directory, processBuilderInternal)
 
-        logDebug("Executing: $cmd")
+        logDebug("Executing: $cmd with timeout $timeOutMillis")
         val status = try {
             processBuilderInternal.exec(cmd, timeOutMillis)
         } catch (ex: ProcessTimeoutException) {
+            logWarn("Timeout reached!")
```

Comment:
I believe it's redundant, because the exception will be rethrown, handled and logged on a higher level

Reply:
Ok, agree